packages/redux/src/settings/reducer/index.ts

```diff
@@ -0,0 +1,7 @@
+import configurationsReducer, { entitiesMapper } from './configurations';
+
+export const entitiesMapperConfigurations = {
```

Comment:
Can you rename it to that, please? This is to keep consistency.

```suggestion
export const configurationsEntitiesMapper = {
```